frontend/src/routes/BillabilityMonthlyWorkingHours.jsx

```diff
@@ -22,14 +24,23 @@ const BillabilityMonthlyWorkingHours = () => {
 
   const stackKeys = ["billable", "non_billable"];
 
+  const { user, loading } = useContext(AuthContext);
+  const navigate = useNavigate();
+
   const onYearChange = (e) => {
     setSelectedYear(e.target.value);
   };
 
-  const applyFilters = () => {
+  const applyFilters = async () => {
     axios
       .get(
-        `http://localhost:3000/billability-working-hours/${selectedYear}/${selectedYearsBack}`
+        `http://localhost:3000/billability-working-hours/${selectedYear}/${selectedYearsBack}`,
+        {
+          headers: {
+            "Content-Type": "application/json",
+            Authorization: `Bearer ${user.accessToken}`,
+          },
```

Comment:
There could be a shared function, `getHeaders(user)` to generate the headers, based on user object.